doc/source/ray-overview/installation.rst

```diff
@@ -3,13 +3,16 @@
 Installing Ray
 ==============
 
-Ray currently supports Linux, MacOS and Windows.
+Ray currently officially supports x86_64 hardware.
```

Comment:
We also supports mac M1 in pypi packages.

Reply:
+1